Fix the duplication of secondary title.

RoySegall opened the pull request:
#7316

I need to check more publication types I mentioned in the issue:

* [x] American Psychological Association 6th Edition
* [x] Chicago Manual of Style (Full Note with Bibliography)
* [x] Chicago Manual of Style (Note with Bibliography)
* [x] Chicago Manual of Style 16th edition (author-date)
* [x] Modern Humanities Research Association (Note with Bibliography)

Before:
![selection_077](https://cloud.githubusercontent.com/assets/1222368/9440174/c1a0414c-4a75-11e5-8698-534141e2b29f.png)

After:
![selection_078](https://cloud.githubusercontent.com/assets/1222368/9440179/c814fa72-4a75-11e5-8b67-ffcd382ef2e3.png)

RoySegall commented:
@rbran100 Totally nailed it! In the csl field map the colletion-title was mapped to `biblio_secondary_title`. Setting it as NULL, like in your query, got it fixed.

rbran100 commented:
@RoySegall will wait to put this in QA till after the additional reference to collection-title is removed

RoySegall commented:
@rbran100 I removed any remaining for the collection-title. I also removed `collection-title-journal` which I suspicions about it but if there any problem i'll bring it back.

rbran100 commented:
@RoySegall looks good, waiting on tests

rbran100 commented:
@RoySegall needed to update the version number of the update, tests are going again

RoySegall commented:
I had a strong felling about it. the tests are green.
